tests/runner.ts

```diff
@@ -79,10 +85,14 @@ function* genTestData(): IterableIterator<TestData> {
         '.extracted',
         testFileEnt.name.replace(/\.json$/, ''),
       );
+
       const outputPath =
-        testData.pluginOptions && testData.pluginOptions.outputPath
-          ? path.join(extractionDir, testData.pluginOptions.outputPath)
-          : path.join(extractionDir, 'translations.{{ns}}.{{locale}}.json');
+        testData.pluginOptions &&
```

Comment:
Could you refactor this to use conditionals instead of ternaries here? Quite hard to follow the nested ternaries.

Reply:
> Thanks. PR looks clean and simple, congratulations. Could you update the documentation? I think you only need to update [this file](https://github.com/gilbsgilbs/babel-plugin-i18next-extract/blob/d680b407ebe9879b5a31f44453d01437ca007353/docs/templates/configuration.md.yml#L181). Please specify that you need a JS babel config for this to work.

going to :)

Reply:
@gilbsgilbs done :)